content/blog/Praezisierung-antrag.de.md

```diff
@@ -0,0 +1,110 @@
+
+## III.	Obliegenheit der betroffenen Person
+
+Zunächst ist es wichtig zu klären, ob die betroffene Person nach Verlangen des Verantwortlichen verpflichtet ist, eine Präzisierung vorzunehmen. Um diese Frage zu beantworten, muss zuerst eine Unterscheidung gemacht werden; Der Auskunftsantrag an eine öffentliche Stelle einerseits und der im Rahmen der Privatwirtschaft.
+
+### 1.	Antrag an öffentliche Stellen
+
+Das Gesetz ist sehr eindeutig in diesen Fällen. Gemäß §34 Abs. 4 BDSG besteht ein solches Recht auf Auskunft nur, soweit die betroffene Person eine Konkretisierung der angeforderten Daten macht. Gleiches gilt für die Fälle der Verarbeitung personenbezogener Daten von zuständigen öffentlichen Stellen für die Verhütung, Ermittlung, Aufdeckung, Verfolgung oder Ahndung von Straftaten oder Ordnungswidrigkeiten. Gemäß §57 Abs. 3 BDSG ist in den letztgenannten Fällen von der Auskunftserteilung abzusehen, wenn die betroffene Person diese Konkretisierung nicht macht. In jedem Fall ist der Antrag abzuweisen, wenn die betroffene Person die angeforderten Informationen nicht eindeutig benennt.
+
+### 2.	Antrag im Rahmen der Privatwirtschaft
+
+In der Privatwirtschaft besteht keine explizite Pflicht zur Präzisierung datenschutzrechtlicher Anfragen. Inwieweit sich eine solche Pflicht indirekt aus dem Gesetz ergibt, ist umstritten.
+
+#### a.	Zwingende Präzisierung
+
+Nach einer Meinung muss die betroffene Person die Informationen und die Verarbeitungszwecke klarstellen, für die sie sich interessiert.[^10] Diese Auslegung führt zu einer Verpflichtung der betroffenen Person, dem Verlangen des Verantwortlichen zwingend nachzukommen. Dieser Meinung folgt auch ein Teil der Rechtsprechung. Dies wird damit begründet, dass die betroffene Person beschreiben müsse, auf welche Bereiche bzw. Kategorien sich ihre Anfrage erstreckt[^11], dass sie verpflichtet sei, den Vorschriften des Erwägungsgrundes zu folgen.[^12] Nach dieser Ansicht ist eine Präzisierung zwingend.
+
+#### b.	Fakultative Präzisierung
+
+Nach dieser Auffassung besteht keine Verpflichtung, den Antrag zu konkretisieren und falls diesem Verlangen nicht entsprochen werde, muss der Verantwortliche umfassend Auskunft erteilen.[^13] Diese Meinung geht davon aus, dass die Aufforderung eine Sollvorschrift darstellt und als solche die betroffene Person nicht zu einer Präzisierung zwingt.[^14] Auch dieser Ansicht folgt ein Teil der Rechtsprechung.[^15] Dieser Ansicht folgend, ist eine Präzisierung des Auskunftsverlangens fakultativ.
+
+#### c.	Zwischenfazit
+
+Weder in der Lehre noch in der Rechtsprechung gibt es eine einheitliche Auffassung. Die DSGVO ist neu, weshalb noch viele normative Lücke zu füllen sind. Deshalb sollte die betroffene Person ihr Auskunftsverlangen so weit wie möglich präzisieren, damit sie in jedem Fall den gesetzlichen Anforderungen genügt. Darüber hinaus zeigt dies ihre Bereitschaft zur Zusammenarbeit, womit sie im Einklang mit Treu und Glauben handelt, § 242 BGB.
```

Comment:
```suggestion
Weder in der Lehre noch in der Rechtsprechung gibt es eine einheitliche Auffassung. Die DSGVO ist neu, weshalb noch viele normative Lücken zu füllen sind. Deshalb sollte die betroffene Person ihr Auskunftsverlangen so weit wie möglich präzisieren, damit sie in jedem Fall den gesetzlichen Anforderungen genügt. Darüber hinaus zeigt dies ihre Bereitschaft zur Zusammenarbeit, womit sie im Einklang mit Treu und Glauben handelt, § 242 BGB.
```

Reply:
Ich glaube ich bin hier nicht deiner Meinung. Wir machen das ja auch explizit anders in unseren Anfragetemplates: Wir fragen erstmal grundsätzlich alles an, auch, weil wir davon ausgehen, dass viele gar nicht unbedingt wissen, welche Daten die Verantwortlichen von ihnen gespeichert haben. Ich bin auch nicht der Meinung, dass es sinnvoll ist von vorne herein mit den Verantwortlichen zu kooperieren. Sicherlich beschleunigt das teilweise den Anfragevorgang, aber auf der anderen Seite sehe ich die Pflicht eher bei den Verantwortlichen, effiziente Prozesse zur Datenauskunft zu entwickeln. Ich würde deshalb empfehlen, zunächst grundsätzlich alles anzufragen und darauf zu warten, dass von der Verantwortlichen die Präzisierung "verlangt" wird, wie im Erwägungsgrund beschrieben. Das hat den Vorteil, dass ich so, falls keine Nachfrage kommt, den kompletten Datensatz bekommen sollte. Und natürlich den Nachteil, dass ich eventuell noch auf eine Präzisierung reagieren muss. Vielleicht kannst du einfach diese beiden Positionen in deinem Artikel darstellen. Mir kommt es nur vor dem Hintergrund, dass der Artikel auf unserer Webseite erscheint, komisch vor, dass wir uns sonst nicht an unsere eigene Empfehlung halten würden.